src/pages/Login/LoginPage.tsx

```diff
@@ -0,0 +1,105 @@
+import { Show, createSignal } from "solid-js";
+import { useNavigate } from "@solidjs/router";
+import log from "loglevel";
+import { AppRoutes } from "@/routes";
+import ErrorModal from "@/components/ErrorModal";
+import * as api from "@/api";
+import { AuthError } from "@/api/auth";
+import "./LoginPage.css";
+import { Button, FormControl, FormErrorMessage, FormLabel, HStack, Input, Spinner, VStack } from "@hope-ui/solid";
+import { createForm } from "@felte/solid";
+import * as yup from 'yup';
+import YupPassword from 'yup-password';
+import { InferType, object, string } from "yup";
+import { validator } from "@felte/validator-yup";
+import beatWithFriendsLogoUrl from '@/assets/img/beatWithFriendsLogo.png';
+import useTranslation from "@/i18n/useTranslation";
+
+YupPassword(yup);
```

Comment:
Do we want to explicitly call this every time? Or just call it in one place?
What happens with TS? How does it know that the objects have been extended?

Reply:
@aburd 
For this PR I only focused on the Login page.
If we start working on a signup page, then it would be better to have this run only once at the start of the application.

In terms of Typescript, the plugin extends yup implementation by exposing the password field for string (`string().password()`).